File: Programlar/Task 1/2/2.py
n = int(input())
numbers = [0]*10
test = False
while n:
    numbers[n%10]+=1
    n//=10
else:
    for i in numbers:
        if i%2==1:
            if(test):
                print("No")
                break
            test=True
    else: print("Yes")


# Verilmiş ədədin rəqəmlərinin yerlərini dəyişdikdə polindrom ədəd alınmağını test edən program

File: Programlar/Task 1/2/2.cpp
#include<iostream>
using namespace std;
int main(){
	int n, arr[10] = {0};
	bool test=false;
	cin>>n;
	while(n){
		arr[n%10]++;
		n/=10;
	}
	for(int i=0; i<10; i++){
		if(arr[i]%2==1){
			if(test){
				cout<<"No";
				return 0;
			}
			test=true;
		}
	}
	cout<<"Yes";
	return 0;
}

// Verilmis ededin reqemlerinin yerlerini deyisdikde polindrom eded alinib-alinmadigini test eden program